src/mame/includes/bublbobl.h

```diff
@@ -2,45 +2,60 @@
 // copyright-holders:Chris Moore, Nicola Salmoria
 
 #include "cpu/m6805/m68705.h"
+#include "cpu/m6800/m6801.h"
+#include "cpu/z80/z80.h"
```

Comment:
Does the header actually use these devices?  If not, including them here slows down the parse and makes dependencies more convoluted.  Don't unnecessarily include things.